### ISSUE TITLE
Project Description

### ISSUE THREAD

**Issue by DipandaAser** [Sanix-Darker/aliash#3]
Hello @Sanix-Darker can we have a project description ?

**Comment by Sanix-Darker** [comment]
yeah, ASAP, but's basically it's a search engine for aliases in bash or powershell for doing various stuff !
the frontend is in preact and obviously, the backend is in golang.

![Screenshot from 2022-09-16 13-13-02](https://user-images.githubusercontent.com/22576758/190626790-7e3e40f6-aebc-4c9c-b1ac-3b46928f5162.png)
